app/assets/javascripts/controllers/navigation/active_item_controller.js

```diff
@@ -0,0 +1,36 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class ActiveItemController extends Controller {
+  static targets = ["menu"];
+  static values = {
+    cssClass: { type: String, default: "active" },
```

Comment:
I have a weak preference for setting data-active rather than using a class – I think classes are a poor way of representing state compared to attributes.